kafka-connect-s3/src/test/java/io/confluent/connect/s3/S3SinkConnectorConfigTest.java

```diff
@@ -156,7 +155,7 @@ public String generatePartitionedPath(String topic, String encodedPartition) {
       }
 
       @Override
-      public List<FieldSchema> partitionFields() {
+      public List<Object> partitionFields() {
```

Comment:
Since this method should not be used in the S3 connector, would it be better to throw UnsupportedOperationException just to help reinforce that fact?

Reply:
Good point! Added. Tests remained unaffected locally, which is expected since this method is not used in the connector.

Reply:
Perfect.